src/main/java/bio/terra/service/dataset/flight/create/CreateDatasetJournalEntryStep.java

```diff
@@ -1,47 +1,28 @@
 package bio.terra.service.dataset.flight.create;
 
-import static bio.terra.service.common.CommonFlightUtils.getFlightInformationOfInterest;
-
 import bio.terra.common.iam.AuthenticatedUserRequest;
 import bio.terra.service.auth.iam.IamResourceType;
+import bio.terra.service.common.JournalRecordCreateEntryStep;
 import bio.terra.service.dataset.flight.DatasetWorkingMapKeys;
 import bio.terra.service.journal.JournalService;
 import bio.terra.stairway.FlightContext;
 import bio.terra.stairway.FlightMap;
-import bio.terra.stairway.Step;
-import bio.terra.stairway.StepResult;
-import bio.terra.stairway.exception.RetryException;
-import java.util.Optional;
 import java.util.UUID;
 
-public class CreateDatasetJournalEntryStep implements Step {
-  private final AuthenticatedUserRequest userReq;
-  JournalService journalService;
-  private UUID journalEntryKey;
+public class CreateDatasetJournalEntryStep extends JournalRecordCreateEntryStep {
 
   public CreateDatasetJournalEntryStep(
-      JournalService journalService, AuthenticatedUserRequest userReq) {
-    this.journalService = journalService;
-    this.userReq = userReq;
+      JournalService journalService,
+      AuthenticatedUserRequest userReq,
+      IamResourceType resourceType,
```

Comment:
It looks like this parameter is always `IamResourceType.DATASET`. If so, does it need to be passed in?